fixtures/worker-app/tests/index.test.ts

```diff
@@ -35,14 +35,22 @@ describe("'wrangler dev' correctly renders pages", () => {
 
 		// Check logged strings are source mapped
 		expect(output).toMatch(
-			/Error: logged error one.+fixtures\/worker-app\/src\/log.ts:7:14/s
+			/Error: logged error one.+fixtures\/worker-app\/src\/log\.ts:7:14/s
 		);
 		expect(output).toMatch(
-			/Error: logged error two.+fixtures\/worker-app\/src\/log.ts:8:14/s
+			/Error: logged error two.+fixtures\/worker-app\/src\/log\.ts:8:14/s
 		);
 		expect(output).toMatch(
-			/Error: logged error three.+fixtures\/worker-app\/src\/log.ts:9:23/s
+			/Error: logged error three.+fixtures\/worker-app\/src\/log\.ts:9:23/s
 		);
```

Comment:
I – and this could just be me – lost context of what we were matching here with the `.+` in the middle

I think this might be clearer:
```
expect(output).toMatch(
    /Error: logged error one\n.+at logErrors.+fixtures\/worker-app\/src\/log\.ts:7:14/
);
expect(output).toMatch(
    /Error: logged error two\n.+at logErrors.+fixtures\/worker-app\/src\/log\.ts:8:14/
);
expect(output).toMatch(
    /Error: logged error three\n.+at logErrors.+fixtures\/worker-app\/src\/log\.ts:9:23/
);
```